src/material/progress-bar/progress-bar.md

```diff
@@ -20,17 +20,16 @@ not necessary to indicate how long it will take should use the indeterminate ind
 In this mode the `value` property is ignored.
 
 #### Buffer
-Operations where the user wants to indicate some activity or loading from the server, 
-use the buffer indicator.
+Use the buffer indicator when want to indicate some activity or loading from the server.
 
 <!-- example(progress-bar-buffer) -->
 
 In "buffer" mode, `value` determines the progress of the primary bar while the `bufferValue` is 
 used to show the additional buffering progress.
 
 #### Query
-For situations where the user wants to indicate pre-loading (until the loading can actually be made), 
-use the query indicator.
+Use the query indicator to indicate pre-loading before the actual loading starts.
```

Comment:
"Indicator to indicate" sounds a little weird. Maybe "Use the `query` mode of the progress bar to indicate..."?

Reply:
This sounds way better :)